src/server/core/journey/route/graph.ts

```
@@ -37,19 +44,26 @@ export interface NodeData {
 }
 
+/**
+ * Get a transit graph for routing between an origin and destination
+ */
 export function getTransitGraph(
   origin: Coordinates,
   destination: Coordinates,
   customMaxOriginWalking?: number,
-  customMaxDestinationWalking?: number
+  customMaxDestinationWalking?: number,
+  includeAccessPoints: boolean = true
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)